jfrog-applications/frogbot/get-started/README.md

```diff
@@ -1,19 +1,18 @@
 # Get Started
 
-JFrog Frogbot is a Git bot that scans your git repositories for security vulnerabilities.
+JFrog Frogbot is a Git bot that scans your git repositories for security vulnerabilities through the following key functions:
 
-1. It scans pull requests immediately after they are opened but before they are merged. This process notifies you if the pull request is about to introduce new vulnerabilities to your code. This unique capability ensures that the code is scanned and can be fixed even before vulnerabilities are introduced into the codebase.
-2. It scans the Git repository periodically and creates pull requests with fixes for vulnerabilities that are detected.
+1. Scans repository selected branches periodically and creates pull requests with fixes for vulnerabilities that are detected.
+2. Scans pull requests making sure no new vulnerabilities are about to enter to your code. 
 
-It supports the following Git providers:
-
-* Azure Repos
-* Bitbucket Server
-* GitHub
-* GitLab
 
 #### Why use JFrog Frogbot?
 
+Regularly scanning your primary branches helps to guarantee the absence of newly discovered vulnerabilities in your existing dependencies.
+Combined with Scan Pull request, It provides comprehensive security coverage for your Git repository utilizing JFrog Security capabilities, which includes:
```

Comment:
I think this phrase is wrong, because scanning repositories doesn't only scan the project dependencies. IaC for example doesn't scan depedencies, and soon other scanners will be added as well.
I recommend removing this entire phrase.

Reply:
I'm trying to think of an answer to the section question, why use Frogbot?
currently the answer is all the scan types. but maybe we can rephrase this trying to emphasize what using Frogbot in general benefits the user.

My initial intention was to give overall view of the security approach we are facing here, meaning scanning your code each day + scan every pull request that is about to enter.